### PR TITLE
QT: Adds descriptions to OSD/Post Processing Settings

### PR DESCRIPTION
**Pull request by Daisouji**
### Rationale behind Changes
Just giving the user more information about the OSD/Post Processing Options. 

### Suggested Testing Steps
Check if the additional descriptions are in proper form.

### CODE REVIEW

File: pcsx2-qt/Settings/GraphicsSettingsWidget.cpp [hunk 1]
```diff
@@ -155,6 +155,20 @@ GraphicsSettingsWidget::GraphicsSettingsWidget(SettingsDialog* dialog, QWidget*
 		tr("Shows the current emulation speed of the system in the top-right corner of the display as a percentage."));
 	dialog->registerWidgetHelp(m_ui.osdShowResolution, tr("Show Resolution"), tr("Unchecked"),
 		tr("Shows the resolution of the game in the top-right corner of the display."));
+	dialog->registerWidgetHelp(m_ui.osdShowCPU, tr("Show CPU Usage"), tr("Unchecked"),
+		tr("Shows workload percentage of host's CPU. Not related to performance outside of PCSX2."));
```
Comment (F0bes) [inline pcsx2-qt/Settings/GraphicsSettingsWidget.cpp:159]: I'd make it simple.
```suggestion
		tr("Shows workload percentage of host's CPU."));
```

File: pcsx2-qt/Settings/GraphicsSettingsWidget.cpp [hunk 2]
```diff
@@ -155,6 +155,20 @@ GraphicsSettingsWidget::GraphicsSettingsWidget(SettingsDialog* dialog, QWidget*
...
+	dialog->registerWidgetHelp(m_ui.osdShowGPU, tr("Show GPU Usage"), tr("Unchecked"),
+		tr("Shows workload percentage of host's GPU. Not related to performance outside of PCSX2."));
```
Comment (F0bes) [inline pcsx2-qt/Settings/GraphicsSettingsWidget.cpp:161]:
```suggestion
		tr("Shows workload percentage of host's GPU."));
```

**Review by F0bes — COMMENTED** [review]
Mentioning that it's only for PCSX2 is purposeless in my opinion. If you wish to however, I'd do something like "Shows host's CPU utilization from PCSX2."

**Comment by Daisouji** [comment]
> Mentioning that it's only for PCSX2 is purposeless in my opinion. If you wish to however, I'd do something like "Shows host's CPU utilization from PCSX2."

Thanks for the suggestion, whatever is simplier.

**Review by JordanTheToaster — APPROVED** [review]

**Comment by refractionpcsx2** [comment]
Can we say "utilization" instead of "workload"? Thanks :)

File: pcsx2-qt/Settings/GraphicsSettingsWidget.cpp [hunk 3]
```diff
@@ -155,6 +155,20 @@ GraphicsSettingsWidget::GraphicsSettingsWidget(SettingsDialog* dialog, QWidget*
 		tr("Shows the current emulation speed of the system in the top-right corner of the display as a percentage."));
 	dialog->registerWidgetHelp(m_ui.osdShowResolution, tr("Show Resolution"), tr("Unchecked"),
 		tr("Shows the resolution of the game in the top-right corner of the display."));
+	dialog->registerWidgetHelp(m_ui.osdShowCPU, tr("Show CPU Usage"), tr("Unchecked"),
+		tr("Shows host's CPU utilization.));
+	dialog->registerWidgetHelp(m_ui.osdShowGPU, tr("Show GPU Usage"), tr("Unchecked"),
+		tr("Shows host's GPU utilization.));
```
Comment (refractionpcsx2) [inline pcsx2-qt/Settings/GraphicsSettingsWidget.cpp:161]: missing closing quote

**Comment by Daisouji** [comment]
Fixed Quotes

**Review by F0bes — APPROVED** [review]

**Review by Mrlinkwii — APPROVED** [review]

**Pull request merged**